src/status_im/activity_center/core.cljs

```diff
@@ -45,38 +53,69 @@
     {:db (update-in db [:activity-center :notifications]
                     update-notifications new-notifications)}))
 
-;;;; Contact verification
-
-(fx/defn contact-verification-decline
-  {:events [:activity-center.contact-verification/decline]}
-  [_ contact-verification-id]
-  {::json-rpc/call [{:method     "wakuext_declineContactVerificationRequest"
-                     :params     [contact-verification-id]
-                     :on-success #(rf/dispatch [:activity-center.contact-verification/decline-success %])
-                     :on-error   #(rf/dispatch [:activity-center.contact-verification/decline-error contact-verification-id %])}]})
-
-(fx/defn contact-verification-decline-success
-  {:events [:activity-center.contact-verification/decline-success]}
+(fx/defn notifications-reconcile-from-response
+  {:events [:activity-center/reconcile-notifications-from-response]}
   [cofx response]
   (->> response
        :activityCenterNotifications
        (map data-store.activities/<-rpc)
        (notifications-reconcile cofx)))
 
-(fx/defn contact-verification-decline-error
-  {:events [:activity-center.contact-verification/decline-error]}
-  [_ contact-verification-id error]
-  (log/warn "Failed to decline contact verification"
-            {:contact-verification-id contact-verification-id
-             :error                   error})
-  nil)
+;;;; Contact verification
+
+(fx/defn contact-verification-decline
+  {:events [:activity-center.contact-verification/decline]}
+  [_ notification-id]
+  {::json-rpc/call [{:method     "wakuext_declineContactVerificationRequest"
+                     :params     [notification-id]
+                     :on-success #(rf/dispatch [:activity-center/reconcile-notifications-from-response %])
+                     :on-error   #(rf/dispatch [:activity-center/process-notification-failure
+                                                notification-id
+                                                :contact-verification/decline
+                                                %])}]})
+
+(fx/defn contact-verification-reply
+  {:events [:activity-center.contact-verification/reply]}
+  [_ notification-id reply]
+  {::json-rpc/call [{:method     "wakuext_acceptContactVerificationRequest"
+                     :params     [notification-id reply]
+                     :on-success #(rf/dispatch [:activity-center/reconcile-notifications-from-response %])
```

Comment:
not really related to this PR, but I'm wondering whether having some function which will receive the event vector and will append results before actually calling `rf/dispatch` will save some `#` and `%` for as. So that we could write something like 
`(rf/dispatch-cb [:activity-center/reconcile-notifications-from-response])`
instead of
`#(rf/dispatch [:activity-center/reconcile-notifications-from-response %])`

Reply:
On top of that @rasom, describing effects as raw data helps with inspecting values (e.g. with `tap>` and with testing (it's easier to match against a vector instead of an anonymous function).